network.go

```diff
@@ -138,6 +170,44 @@ func (n *VirNetwork) GetXMLDesc(flags uint32) (string, error) {
 	return xml, nil
 }
 
+func (n *VirNetwork) UpdateXMLDesc(xmldesc string, command, section int) error {
+	xmldescC := C.CString(xmldesc)
+	result := C.virNetworkUpdate(n.ptr, C.uint(command), C.uint(section), C.int(-1), xmldescC, C.uint(C.VIR_NETWORK_UPDATE_AFFECT_CURRENT))
+	C.free(unsafe.Pointer(xmldescC))
+	if result == -1 {
+		return GetLastError()
+	}
+	return nil
+}
+
+func getHostXMLDesc(ip, mac, name string) string {
+	var b bytes.Buffer
+	b.WriteString("<host ")
+	if len(ip) > 0 {
+		b.WriteString(fmt.Sprintf(" ip=\"%s\"", ip))
+	}
+	if len(mac) > 0 {
+		b.WriteString(fmt.Sprintf(" mac=\"%s\"", mac))
+	}
+	if len(name) > 0 {
+		b.WriteString(fmt.Sprintf(" name=\"%s\"", name))
+	}
+	b.WriteString(" />")
+	return b.String()
+}
```

Comment:
What about using XML marshal for that? That would let me sleep tight at night :)

I can take care of that of course